app/graphql/mutations/base_mutation.rb

```diff
@@ -30,6 +30,20 @@ def attribute_required?(attribute)
 
     def authorize!(resource, action)
       user = context[:current_user]
+      if resource.class.included_modules.include?(Authorizable)
```

Comment:
settings do not have permissions, only admin can change them

Reply:
There are actually `view_settings` and `edit_settings` permissions, although it looks like they were not fully implemented - they are only used to limit access to the endpoints and in the ui

Reply:
Right, that was for canned admin... lets see how many things will break when I try to make settings authorizable...